packages/components/.storybook/web/preview.tsx

```diff
@@ -22,4 +22,7 @@ export const parameters = {
       return React.createElement(DocsContainer, currentProps)
     },
   },
+  viewport: {
+    defaultViewport: 'mobile2',
+  },
```

Comment:
Unrelated to ticket, but sets default viewport to mobile in Storybook Web. Let me know if you have any issues with this.

Reply:
At face value, I didn't think I'd like it, but seems alright to me.

Only concern is if it might impact the iframes in the doc site. I don't think it will, but something to check once merged.